lib/screens/more_details_page.dart

```diff
@@ -0,0 +1,223 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/painting.dart';
+import 'package:project_eureka_flutter/components/eureka_appbar.dart';
+
+import 'package:project_eureka_flutter/screens/home_page.dart';
+
+class MoreDetails extends StatefulWidget {
+  @override
+  _MoreDetailsState createState() => _MoreDetailsState();
+}
+
+class _MoreDetailsState extends State<MoreDetails> {
+  String id; // user id
+
+  @override
+  void initState() {
+    initGetQuestionDetails(id);
+    super.initState();
+  }
+
+  void initGetQuestionDetails(String id) {}
+
+  Column _headerStack() {
+    return Column(
+      children: [
+        Container(
+          height: 0,
+        ),
+        _profileNameAndIcon(),
+        _answerQuestion(),
+      ],
+    );
+  }
+
+  Row _profileNameAndIcon() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.start,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        CircleAvatar(
+          radius: 55.0,
+          backgroundColor: Colors.grey[300],
+          child: CircleAvatar(
+            radius: 40.0,
+            backgroundImage: AssetImage('assets/images/dn.jpg'),
+          ),
+        ),
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              "Question Title here",
+              style: TextStyle(fontSize: 25.0, fontWeight: FontWeight.bold),
+              textAlign: TextAlign.left,
+            ),
```

Comment:
Question title is too close to the edge of the  container. Consider adding a `SizedBox(height:  );` on line 52.

Reply:
I agree with Jane, also if the title is longer than certain amount of characters, it will cause an error. It must be more flexible.

Reply:
I think the question title should be above the this Widget's return Row, that way the title will be above the profile icon and it can wrap itself to a new line if it needs room.

Reply:
done